tools/docker/docker-compose.yml

```diff
@@ -14,12 +14,30 @@ services:
             MYSQL_DATABASE: ragnarok
             MYSQL_USER: ragnarok
             MYSQL_PASSWORD: ragnarok
-    server:
+
+    builder:
         image: "rathena:local"
-        container_name: "rathena"
+        container_name: "rathena-builder"
+        command: "/rathena/tools/docker/builder.sh"
+        volumes:
+            - "../..:/rathena" # mount git repo directory inside container
+            - "./asset/inter_conf.txt:/rathena/conf/import/inter_conf.txt" # load db connection
+            - "./asset/char_conf.txt:/rathena/conf/import/char_conf.txt"   #localdev login-char relation
+            - "./asset/map_conf.txt:/rathena/conf/import/map_conf.txt"     #localdev char-map relation
+        init: true # helps with signal forwarding and process reaping
+        tty: true
+        stdin_open: true
+        build:
+            context: .
+            dockerfile: Dockerfile
+        environment:
+            BUILDER_CONFIGURE: "--enable-packetver=20200401"
```

Comment:
Is there a reason you're defining an old packetver?

Reply:
Not really, I've updated with the one defined in master branch